### DISCUSSION TITLE
I there a way to export and update unit stats?

### DISCUSSION THREAD

**Discussion by Benoss** [data-bomb/Silica#27] [Q&A]
I was wondering what was the easiest way to update unit stats in a mod.
Is there a way to export all of the vanilla stats and modify them via a mod?

**Comment by data-bomb** [comment]
@Benoss there are a fair amount of both legitimate and illegitimate use cases involved with your question. Even if you do have good intentions, it won't be discussed here since the same information could be leveraged to make client-side cheats even easier than they already are to make. You're welcome to join the Discord to discuss the question if you feel this isn't a fair answer.
